rados/rados_test.go

```diff
@@ -1122,6 +1122,31 @@ func (suite *RadosTestSuite) TestOpenIOContextInvalidPool() {
 	require.Nil(suite.T(), ioctx)
 }
 
+func (suite *RadosTestSuite) TestGetLastVersion() {
+	suite.SetupConnection()
+	oid := suite.GenObjectName()
+	v1 := suite.ioctx.GetLastVersion()
+
+	bytes_in := []byte("something to write")
+	err := suite.ioctx.Write(oid, bytes_in, 0)
+	assert.NoError(suite.T(), err)
+
+	v2 := suite.ioctx.GetLastVersion()
+	assert.NotEqual(suite.T(), v1, v2)
+
+	v3 := suite.ioctx.GetLastVersion()
+	assert.Equal(suite.T(), v2, v3)
+
+	bytes_in = []byte("something completely different")
+	err = suite.ioctx.Write(oid, bytes_in, 0)
+	assert.NoError(suite.T(), err)
+
+	v4 := suite.ioctx.GetLastVersion()
+	assert.NotEqual(suite.T(), v1, v4)
+	assert.NotEqual(suite.T(), v2, v4)
+	assert.NotEqual(suite.T(), v3, v4)
+}
```

Comment:
May be we can add one test case for Read() and check whether the version changes with Read() also or not.

Reply:
Sounds reasonable enough. I'll look into it.

Reply:
Tests extended.